### ISSUE TITLE
Invalid transparency (compression off, APNG source is valid)

### ISSUE THREAD

**Issue by GDXRepo** [onevcat/APNGKit#21]
Can't understand the problem. I try to load simple APNG image, it is valid (I've checked it via several browsers). And when I look for it with debug mode enabled, I see normal image too. But on the Simulator and on a real device I see image without transparency, where all semi-transparent pixels are set to white. I can't understand what's the reason of this. Compression is OFF in Xcode, file source has "apng" extension (double checked this). When I tried to load this image via UIImage - I saw the normal picture. What's wrong with me?

P.S. Code is pretty simple.
```
override func viewDidAppear(_ animated: Bool) {
    let image = APNGImage(named: "scene1")
    let imageView = APNGImageView(image: image)
    view.addSubview(imageView)
}
```
Here you can see my application frozen with View Debug mode. Normal image in Debug area and broken inside the Simulator...

<img width="1343" alt="issue" src="https://cloud.githubusercontent.com/assets/11994363/18568756/caad656e-7baf-11e6-86a1-7c68f2102e77.png">

Environment:
Xcode 8 GM Seed
iOS Simulator - iOS 9.3 / 10.0.1
iPhone 6S - iOS 10.0.1

**Comment by GDXRepo** [comment]
No solutions? Or at least advices, what I've done wrong) Maybe invalid images (who knows?) - then give me please any valid animated png, if possible, to check this issue again.

**Comment by onevcat** [comment]
Should be fixed in version 0.3.1 (as well as in swift2.3 branch).
